kivy/input/postproc/calibration.py

```diff
@@ -94,7 +106,32 @@ def process(self, events):
             elif event.ud['calibration:frame'] == frame:
                 continue
             params = self.devices[event.device]
-            event.sx = event.sx * params['xratio'] + params['xoffset']
-            event.sy = event.sy * params['yratio'] + params['yoffset']
+            if 'auto' in params:
+                event.sx, event.sy = self.auto_calibrate(
+                    event.sx, event.sy, params['auto'])
+                if not (0 <= event.sx <= 1 and 0 <= event.sy <= 1):
+                    to_remove.append((etype, event))
+            else:
+                event.sx = event.sx * params['xratio'] + params['xoffset']
+                event.sy = event.sy * params['yratio'] + params['yoffset']
             event.ud['calibration:frame'] = frame
+
+        for event in to_remove:
+            events.remove(event)
+
         return events
+
+    def auto_calibrate(self, sx, sy, size):
+        from kivy.core.window import Window as W
+        WIDTH, HEIGHT = size
```

Comment:
@tshirtman What do you think about using SDL_GetCurrentDisplayMode to fetch the screen size dynamically? 
I found another PR ([here](https://github.com/kivy/kivy/pull/4286/commits/f143512ce3de02f698d72b78d193f56b0d1abe68)) that added the functions to retrieve it, when I patched that into this PR and used those values it worked fine. Then a user only needs to add auto=true to their config, makes it a bit easier.